src/cbc_sdk/platform/processes.py

```diff
@@ -29,29 +48,33 @@
 
 
 class Process(UnrefreshableModel):
-    """Represents a process retrieved by one of the Enterprise EDR endpoints.
+    """
+    Information about a process running on one of the endpoin ts connected to the Carbon Black Cloud.
+
+    Objects of this type are retrieved through queries to the Carbon Black Cloud server, such as via
+    ``AsyncProcessQuery``.
```

Comment:
Seems odd to mention AsyncProcessQuery then have an example using synchronous query

Reply:
I didn't name the class.  Despite the name, `AsyncProcessQuery` can be used synchronously. Maybe the name needs changing?